ports/rp2/CMakeLists.txt

```diff
@@ -146,20 +148,20 @@ set(MICROPY_SOURCE_QSTR
     ${MICROPY_DIR}/shared/readline/readline.c
     ${MICROPY_DIR}/shared/runtime/mpirq.c
     ${MICROPY_DIR}/shared/runtime/sys_stdio_mphal.c
-    ${PROJECT_SOURCE_DIR}/machine_adc.c
-    ${PROJECT_SOURCE_DIR}/machine_i2c.c
-    ${PROJECT_SOURCE_DIR}/machine_i2s.c
-    ${PROJECT_SOURCE_DIR}/machine_pin.c
-    ${PROJECT_SOURCE_DIR}/machine_rtc.c
-    ${PROJECT_SOURCE_DIR}/machine_spi.c
-    ${PROJECT_SOURCE_DIR}/machine_timer.c
-    ${PROJECT_SOURCE_DIR}/machine_uart.c
-    ${PROJECT_SOURCE_DIR}/machine_wdt.c
-    ${PROJECT_SOURCE_DIR}/modmachine.c
-    ${PROJECT_SOURCE_DIR}/modrp2.c
-    ${PROJECT_SOURCE_DIR}/modos.c
-    ${PROJECT_SOURCE_DIR}/rp2_flash.c
-    ${PROJECT_SOURCE_DIR}/rp2_pio.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_adc.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_i2c.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_i2s.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_pin.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_rtc.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_spi.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_timer.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_uart.c
+    ${CMAKE_CURRENT_LIST_DIR}/machine_wdt.c
+    ${CMAKE_CURRENT_LIST_DIR}/modmachine.c
+    ${CMAKE_CURRENT_LIST_DIR}/modrp2.c
+    ${CMAKE_CURRENT_LIST_DIR}/modos.c
+    ${CMAKE_CURRENT_LIST_DIR}/rp2_flash.c
+    ${CMAKE_CURRENT_LIST_DIR}/rp2_pio.c
```

Comment:
Same comment as for esp32 port changes.

Reply:
No matter what happens with ESP32 this will definitely work.  👍